VMEncryption/main/DiskUtil.py

```diff
@@ -319,6 +319,10 @@ def umount(self, path):
         return self.command_executor.Execute(umount_cmd)
 
     def mount_all(self):
+        # Reload systemd daemon to get lastest changes from fstab
+        # pidof systemd seems unreliable on Ubuntu hence directly invoking systemctl
```

Comment:
actual: "pidof"
expected: "pid of"

Reply:
I was trying to use command **pidof systemd** to determine if systemd is in use. But it turns out that this command was not very reliable on Ubuntu. Hence, just wrote the command and reason not to use it here.

Reply:
👍